packages/cozy-client/src/models/fsnative.js

```diff
@@ -1,5 +1,6 @@
-import { isAndroidApp, isIOS } from 'cozy-device-helper'
+import { isMobileApp, isAndroidApp, isIOS } from 'cozy-device-helper'
 
+import CozyClient from '../CozyClient'
```

Comment:
Why this import?

Reply:
Pour le type `@param {CozyClient} client - cozy.client instance`